lib/resin-lint.coffee

```diff
@@ -53,35 +67,58 @@ module.exports = (passed_params) ->
 			.describe('f', 'Specify a coffeelint config file to override resin-lint rules')
 			.describe('p', 'Print default resin-lint coffeelint.json')
 			.describe('i', 'Ignore coffeelint.json files in project directory and its parents')
+			.boolean('u', 'Run unused import check')
 
 		if options.argv._.length < 1 and not options.argv.p
 			options.showHelp()
 			process.exit(1)
 
-		if options.argv.p
-			console.log(fs.readFileSync(CONFIG_PATH).toString())
-			process.exit(0)
+		Promise.try ->
+			if options.argv.u
+				return Promise.map options.argv._, (dir) ->
+					dir = getPackageJsonDir(dir)
+					Promise.resolve depcheck path.resolve('./', dir),
```

Comment:
`getPackageJsonDir` returns either a relative or an absolute path, depending on what the input was. Even so that should be fine shouldn't it? It will just return the current directory:
```javascript
> path.resolve('./', '/home/cameron/test/absolute/path')
'/home/cameron/test/absolute/path'